lib/log_ratelimit.c

```diff
@@ -74,6 +82,9 @@ log_ratelimit_allow(struct log_ratelimit_state *lrs)
 {
 	uint64_t now;
 
+	if (unlikely(!enabled))
```

Comment:
Let's add the following comment for the `if` test::
`/* unlikely() reason: @enabled is only false during startup. */`